config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -243,6 +243,10 @@ config OPENTHREAD_XPANID
     string
     default "11:11:11:11:22:22:22:22"
 
+config IEEE802154_NRF5_RX_STACK_SIZE
```

Comment:
@kkasperczyk-no could you please justify this change?

Reply:
This config is set in every prj.conf, but applies only to 15.4 radio, so Thread. I moved it to the defaults if Thread is enabled, so it could be removed from prj.confs.

Reply:
But shouldn't we remove it from general CHIP defaults (non Thread specific) in this case? I mean now this config appears twice in this Kconfig.defaults file. Or am I missing something?